webapp/views/App/views/Designer/TaxonomyList/Row/Row.js

```diff
@@ -0,0 +1,97 @@
+import './Row.scss'
+import * as A from '@core/arena'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import * as Survey from '@core/survey/survey'
+import * as Taxonomy from '@core/survey/taxonomy'
+
+import { useI18n, useLang } from '@webapp/store/system'
+import { useSurvey, useSurveyInfo } from '@webapp/store/survey'
+import { useAuthCanEditSurvey } from '@webapp/store/user'
+
+import ErrorBadge from '@webapp/components/errorBadge'
+import WarningBadge from '@webapp/components/warningBadge'
+
+import { State, useTaxonomyRow } from './store'
+
+const TableButton = ({ show = true, label, onClick, icon }) => {
```

Comment:
I dont think this component saves some line of code

Reply:
also, if you return null, you might get in trouble with the grid-template-columns